conf/messages.en

```diff
@@ -171,4 +172,10 @@ claimPeriod.Oct2021 = October 2021
 claimPeriod.Nov2021 = November 2021
 claimPeriod.Dec2021 = December 2021
```

Comment:
why are these here?

Reply:
these are messages displayed as radio buttons

Reply:
we only need Nov2021 for now in that case